src/generate/handlebarsAmfHelpers.ts

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  getPayloadResponses,
+  extractTypeFromPayload,
+  getDataType,
+  getPayloadType,
+  getFilteredProperties,
+  getValue,
+  DEFAULT_DATA_TYPE,
+  ARRAY_DATA_TYPE,
+  OBJECT_DATA_TYPE
+} from "./utils";
+
+import { model } from "amf-client-js";
+
+/**
+ * Selects the baseUri from an AMF model. TypeScript will not allow access to
+ * the data without the proper cast to a WebApi type.
+ *
+ * @param property - A model from the the AMF parser
+ *
+ * @returns the base URI of the model
+ */
+export const getBaseUri = (
+  property: model.document.BaseUnitWithEncodesModel
+): string => {
+  return property && property.encodes
+    ? (property.encodes as model.domain.WebApi).servers[0].url.value()
+    : "";
+};
+
+/**
+ * Checks the node is a type definition.
+ *
+ * @param obj - The node to check
+ *
+ * @returns true if the node is a type definition, false if not
+ */
+export const isTypeDefinition = (
+  cdProperty: model.domain.DomainElement
```

Comment:
What does cdProperty stand for?

Reply:
This type was CustomDomainProperty earlier and I changed it to DomainElement later but forgot to change the name. Will change it.